Sources/JavaKitJar/generated/Attributes.swift

```diff
@@ -95,3 +98,92 @@ public struct Attributes {
   @JavaMethod
   public func getOrDefault(_ arg0: JavaObject?, _ arg1: JavaObject?) -> JavaObject?
 }
+extension Attributes {
+  @JavaClass("java.util.jar.Attributes$Name")
+  public struct Name {
```

Comment:
This is a wonderful improvement, but I think it should be driven by adding

```
"java.util.jar.Attributes$Name" : "Attributes.Name",
```

to this module's `Java2Swift.config`.